RollGen.Domain/DomainDice.cs

```diff
@@ -49,7 +49,7 @@ public string ReplaceWrappedExpressions<T>(string source, string expressionOpen
                 var lenientReplacedDice = ReplaceDiceExpression(matchGroupValue, true);
                 var unevaluatedMatch = Evaluate(lenientReplacedDice);
                 var target = expressionOpen + matchGroupValue + expressionClose;
-                var replacement = BooleanOrType<T>(unevaluatedMatch).ToString();
+                var replacement = Utils.BooleanOrType<T>(unevaluatedMatch).ToString();
```

Comment:
Do we/you have use cases where one wants to do those conversions?  Personal taste tends to dictate to make as many things private/hidden/internal as possible, and only show when there is a confirmed use case that necessitates it.  So if you already have a use case where being able to access those conversions helps, then great.  Otherwise, let's keep them hidden.